packages/docs/docs/docs/zeptoken.md

```diff
@@ -0,0 +1,38 @@
+---
+id: zeptoken
+title: The ZEPToken
+sidebar_label: The ZEPToken
+---
+
+ZeppelinOS' native ZEP token is used to incentivize the creation and maintenance of a healthy ecosystem of EVM packages. It does so via a series of mechanisms that are detailed in the following sections of this article.
+
+## Compatibility standards
+
+ZEP is an ERC20 token, controlled by a [TPL](https://github.com/TPL-protocol) (Transaction Permission Layer) jurisdiction. This allows regional jurisdictions to control which addresses can hold or exchange ZEP using KYC, or whatever other validation process are found to be adequate for the region.
+
+The ZEP token's source code can be found at [`zos/packages/vouching/contracts/ZEPToken.sol`](https://github.com/zeppelinos/zos/blob/master/packages/vouching/contracts/ZEPToken.sol).
+
+## EVM package vouching
+
+Anyone can create ZeppelinOS EVM packages for free. More precisely, you don't need to hold or spend ZEP to create an EVM package. 
+
+However, ZeppelinOS provides a mechanism in which an EVM package can be registered and vouched for using ZEP. The collected stake of an EVM package can be challenged by other ZEP holders whenever a deficiency in the EVM package is presented for evaluation. In such a situation, the package's stake could be slashed in favor of the challenger.
+
+This simple mechanism allows ZEP that is vouched for an EVM package to represent:
```

Comment:
It looks odd to list a set of features here, only to have a following paragraph pointing out which is the _actual_ set of features. I'd reprhase this sentence mentioning that the _end goal_ for the ZEP token vouching is x, y, and z, to then mention the actions available now.